packages/medusa/src/loaders/api.ts

```diff
@@ -52,6 +50,7 @@ export default async ({
       throw Error("An error occurred while registering Medusa Core API Routes")
     }
   } else {
+    app.use(bodyParser.json())
```

Comment:
Q: wouldn't this break the v1 webhooks?

Reply:
I mean that the webhook in v1 will have the body parsed no?

Reply:
No since in the plugins loader we only call `RoutesLoader` which will apply body parser depending on the config